src/commands/synthetics/interfaces.ts

```diff
@@ -380,6 +380,13 @@ export interface CommandConfig extends SyntheticsCIConfig {
   failOnTimeout: boolean
 }
 
+export interface UploadApplicationCommandConfig extends SyntheticsCIConfig {
+  mobileApplicationVersionFilePath?: string
+  mobileApplicationId?: string
+  versionName?: string
+  latest?: boolean
+}
+
```

Comment:
Rather than extending `SyntheticsCIConfig` with a lot of unnecessary fields, let's just copy the fields that are required for this new command. And we can create a card to refactor these interfaces to avoid the duplication.